writer/template.go

```diff
@@ -0,0 +1,30 @@
+package writer
+
+import (
+	"io"
+	"text/template"
+
+	"github.com/haya14busa/errorformat"
+)
+
+// Template represents text/template based writer.
+type Template struct {
+	tmpl *template.Template
+	w    *trackingWriter
+}
+
+func NewTemplate(tmpl *template.Template, w io.Writer) *Template {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function NewTemplate should have comment or be unexported